static/app/components/issues/groupListHeader.tsx

```diff
@@ -4,25 +4,36 @@ import {PanelHeader} from 'sentry/components/panels';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 
+import {GroupListColumn} from './groupList';
+
 type Props = {
   withChart: boolean;
   narrowGroups?: boolean;
   showLastTriggered?: boolean;
```

Comment:
and we'd want to get rid of the unused prop

Reply:
updated